cmd/end2endtest/main.go

```diff
@@ -110,6 +110,10 @@ func main() {
 	flag.CommandLine.SortFlags = false
 	flag.Parse()
 
+	if c.parallelCount > c.nvotes {
+		c.parallelCount = c.nvotes
+	}
+
```

Comment:
this is actually an unrelated fix but in this case it was me :sweat_smile: if you make a separate PR ("end2endtest small fixes") i'll move it there

Reply:
along with voteOverwrite fixes and so on